esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1122,7 +1178,7 @@ unsigned WsWuInfo::getWorkunitThorLogInfo(IArrayOf<IEspECLHelpFile>& helpers, IE
                     continue;
 
                 const char* pStr = logName.str();
-                const char* ppStr = strstr(pStr, "/thormaster.");
+                const char* ppStr = strstr(pStr, thorMasterLogDateSearchString.get());
```

Comment:
why is this necessary?
To get these logs, it has called cw->getLogs("Thor", ...)
so they explicitly thor log files already (no matter what name they have)

Reply:
Also note:
```
ppStr += 12
```
would be wrong if thorMasterLogDateSearchString varies. Should avoid constants like this

Reply:
The code here is to find out the log date. Some WU may run for > 1 days. ECLWatch has to display a dialog box for a user to choose a thor slave using the log date, as well as thor process name and a slave ID.

Reply:
But should not use constants like '12'.
Use a const string and strlen if necessary for readability/maintainability.